index.html

```diff
@@ -703,6 +758,19 @@ <h2><a href="#events" class="anchor">🔗</a> Events</h2>
             </a>
           </small>
         </li>
+        <li>
+          <a href="https://www.w3.org/2014/03/lgd/papers/lgd14_submission_68">
+            <q>GeoGratis API</q> reference
+          </a> and
+          <a href="https://www.w3.org/community/maps4html/wiki/LGD14BarCampNotes">BarCamp notes</a>
+          – extending the web to enable linked geospatial data (birth of the Maps for HTML Community Group!)
```

Comment:
The descriptions here may be improved, but as long as they're not inaccurate I'm happy. :)